skeletons/constr_CHOICE_jer.c

```diff
@@ -249,12 +249,12 @@ CHOICE_decode_jer(const asn_codec_ctx_t *opt_codec_ctx,
         }
 
         ASN_DEBUG("Unexpected JSON key [%c%c%c%c] in CHOICE [%s]"
-                  " (ph=%d, key=%s)",
+                  " (ph=%d)",
                   ch_size>0?((const uint8_t *)buf_ptr)[0]:'?',
                   ch_size>1?((const uint8_t *)buf_ptr)[1]:'?',
                   ch_size>2?((const uint8_t *)buf_ptr)[2]:'?',
                   ch_size>3?((const uint8_t *)buf_ptr)[3]:'?',
-                  td->name, ctx->phase, json_key);
+                  td->name, ctx->phase);
```

Comment:
If you find the key unexpected - wouldn't it make sense to print out what the offending key is?

Reply:
The wording is wrong, should be a general set of JSON characters, or "token", which is not handled by the previous logic. It is not just a "key".